front-end/src/app/shared/models/transaction-groups/transaction-group-l.model.ts

```diff
@@ -0,0 +1,57 @@
+import { LabelUtils, PrimeOptions } from 'app/shared/utils/label.utils';
+import { ContactTypeLabels, ContactTypes } from '../contact.model';
+import { TransactionTemplateMapType } from '../transaction-type.model';
+import { TransactionGroup } from './transaction-group.model';
+
+export class TransactionGroupL extends TransactionGroup {
+  getFormProperties(templateMap: TransactionTemplateMapType): string[] {
+    return [
+      'entity_type',
+      templateMap.organization_name,
```

Comment:
Looks like group L is supposed to be broader (IND,ORG,COM) with a little snag where Contribution_to_candidate_in_kind won't allow COM.  But it's also not applicable to this ticket anymore because the transactions in this ticket are now group M